tests/system/src/test/java/test/com/sun/javafx/application/InitializeJavaFXStartupTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2012, 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package test.com.sun.javafx.application;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class InitializeJavaFXStartupTest extends InitializeJavaFXBase {
+
+    @BeforeClass
+    public static void initialize() throws Exception {
+        System.out.println("Calling Startup!");
+        InitializeJavaFXBase.initializeStartup();
+        System.out.println("Called Startup!");
+    }
+
+    @Test (timeout = 15000)
+    public void testStartupThenLaunchInFX() throws Exception {
+        doTestInitializeThenLaunchInFX();
+    }
+
+    @Test (timeout = 15000)
+    public void testStartupThenLaunch() throws Exception {
```

Comment:
It will be more robust for these two test methods to be in separate files. Otherwise they might interfere with each other. If `testStartupThenLaunchInFX` runs first, then an error in that test (e.g., if the fix isn't applied and it times out) will cause the `testStartupThenLaunch` to fail. Conversely, if `testStartupThenLaunch` runs first, then the system is in a state where the Application has already been started by the time `testStartupThenLaunchInFX` runs. It will still pass, but won't be a solid test of the fix, since that case would fail today.

Reply:
They are seperated now!